tests/unit/test_sync.py

```diff
@@ -135,12 +136,14 @@ def test_sync_acoustic_record_delete_path(
         "end_time": end_time.isoformat(),
     }
     if no_metrics:
-        assert watcher.count == 4  # Get All Records, Get Contact(x2), Increment Retry
+        assert (
+            watcher.count == 5
+        )  # Get Acoustic Fields, All Records, Get Contact(x2), Increment Retry
         assert context == expected_context
         return
 
     # Metrics adds two DB queries (total records and retries)
-    assert watcher.count == 6
+    assert watcher.count == 7
```

Comment:
Is it worth having assertions like this, or is this something we can safely delete?

Reply:
I added an operation that introduces a new SQL statement. Therefore I incremented the value and modified the code comment.

We could invest some time in the unit tests, but I personally would prefer if we wouldn't block this PR for that